.github/workflows/test.yml

```diff
@@ -25,6 +25,8 @@ jobs:
             autopilot
             bus bus/client
             worker worker/client
+          flags: |
+            -japecheck.types=false
```

Comment:
This is kind of unfortunate. Do we have to do this or could this be a debug leftover? If we have to do it, can we get around doing it by using `jc.Custom` in the required places.

Reply:
`jc.Custom` is pretty annoying and nasty. Better to not use it.

Reply:
Oops. Missed this.  I didn't think it was too bad but I'll undo it

Reply:
Hm ok. I'm a bit surprised we're giving up on type checking only for the sake of same `jc.Custom`'s. I agree they are annoying but at the same time they're not really in the way of anything. Thanks for the link on the discussion, I missed that.

Reply:
I agree it's a bit sad, but we spend a lot of time and effort fighting jape to do normal API things. Adding no-op funcs to satisfy an external analyzer isn't ideal. You can pretty much get the same results with a test, imo.